app/controllers/services_controller.rb

```diff
@@ -17,6 +17,8 @@ def create
   end
 
   def edit
+    @undo_redo_button = session.delete(:undo)
```

Comment:
```suggestion
    if ENV['undo_redo'] == 'enabled'
      @undo_redo_button = session.delete(:undo)
    end
```

assuming you don't want to launch the feature without the error messages?

Reply:
Added feature flag